### PR TITLE
Footer and social links

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Adds `react svg loader` to `webpacker`.

### CODE REVIEW

File: config/webpack/environment.js [hunk 1]
```diff
@@ -1,6 +1,27 @@
 const { environment } = require('@rails/webpacker');
 const path = require('path');
 
+const babelLoader = environment.loaders.get('babel');
+
+environment.loaders.insert(
```
Comment (luistorres) [inline config/webpack/environment.js:6]: This one is much better! Can you put this loader in the firefighters project too, please? (remove the current one) It doesn't have to be now though. In the future!!!

Comment (luistorres) [reply config/webpack/environment.js:6]: I saw it but I though it didn't have much support. Yet, it seems to work much better

**Review by luistorres — APPROVED** [review]
🚀 🤶

File: config/webpack/environment.js (hunk 1 above)
Comment (jfranciscosousa) [reply config/webpack/environment.js:6]: Svg inline loader? We used that in the Oncostats

Comment (luistorres) [reply config/webpack/environment.js:6]: Probably

**Pull request merged**